packages/langium/src/services.ts

```diff
@@ -85,7 +86,7 @@ export type LangiumLspServices = {
     SemanticTokenProvider?: SemanticTokenProvider
     RenameProvider?: RenameProvider
     Formatter?: Formatter
-    SignatureHelp?: SignatureHelpProvider
+    SignatureHelpProvider?: SignatureHelpProvider
```

Comment:
This is definitely breaking, I'm not sure how to handle it in the 1.x version stream.

Reply:
I've reverted the change.